pkg/appfile/storage/storage.go

```diff
@@ -1,37 +1,44 @@
 package storage
 
 import (
+	"errors"
+	"os"
+
 	"github.com/oam-dev/kubevela/pkg/appfile/storage/driver"
 )
 
 // Store application store client
 var Store *Storage
 
-func init() {
-	// TODO support provide multiple ways:
-	// system environment
-	// system configfile
-	// startup arguments
-	Store = NewStorage(driver.LocalDriverName)
-}
-
 // Storage is common storage client，use it to get app and others resource
 type Storage struct {
 	driver.Driver
 }
 
+func init() {
+	// default use Local Storage driver
+	_ = NewStorage(driver.LocalDriverName)
```

Comment:
why ignore the error? This might not work in the controller as an init() function.